runtime/src/migration.rs

```diff
@@ -18,10 +21,24 @@ impl<T: Trait> Module<T> {
         // would be any new storage values that need an initial value which would not
         // have been initialized with config() or build() chainspec construction mechanism.
         // Other tasks like resetting values, migrating values etc.
+
+        Self::initialize_storage_working_group_mint();
+        Self::initialize_storage_working_group_text_constraints();
+
+        // ** Order is important!
+        Self::clear_storage_data_object_registry_data();
+        Self::clear_storage_data_directory_data();
```

Comment:
maybe both steps in a single `fn clear_storage_data()` and avoid need for the "order is important" comment ?

How about also resetting the `next_*_id` state variables to their initial values as well ?

Reply:
Both fixed.